authorization/src/main/java/com/terabite/movement/controller/CategoryController.java

```diff
@@ -14,33 +15,39 @@
 @CrossOrigin(allowCredentials = "true", origins = "http://localhost:3000")
 @RestController
 @RequestMapping("/v1/cat")
```

Comment:
I am having difficulty seeing the need for categories to have their endpoint. 

Could we not just make it part of the movement endpoint to get movements by categories? so instead of having to do `v1/cat/categories/{id}`
you could do `v1/mov/category/{id}`